### PR TITLE
DR2-1236 Add Identifiers if there are any in Dynamo

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: src/test/scala/uk/gov/nationalarchives/LambdaTest.scala [hunk 1]
```diff
@@ -238,12 +268,12 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
...
 
   "handleRequest" should "return an error if the dynamo entry does not have a type of 'asset'" in {
-    stubGetRequest(dynamoGetResponse.replace("Asset", "Folder"))
+    stubGetRequest(dynamoGetResponse.replace("Asset", "ArchiveFolder"))
     stubQueryRequest(emptyDynamoQueryResponse)
     val ex = intercept[Exception] {
       TestLambda().handleRequest(standardInput, outputStream, null)
     }
-    ex.getMessage should equal(s"Object $assetId is of type Folder and not 'asset'")
+    ex.getMessage should equal(s"Object $assetId is of type ArchiveFolder and not 'asset'")
```
Comment (techncl) [inline src/test/scala/uk/gov/nationalarchives/LambdaTest.scala:276]: Should "asset" here and in the test name be capitalised?

File: src/test/scala/uk/gov/nationalarchives/LambdaTest.scala [hunk 2]
```diff
@@ -320,6 +350,22 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
     contentObjects.last.text should equal(childIdJson.toString)
   }
 
+  "handleRequest" should "write the opex file to s3" in {
```
Comment (techncl) [inline src/test/scala/uk/gov/nationalarchives/LambdaTest.scala:353]: Maybe "Should make an S3 copy request with the correct opex file"?

**Review by techncl — COMMENTED** [review]
Looks good! Just some minor comments.

File: src/test/scala/uk/gov/nationalarchives/LambdaTest.scala (hunk 1 above)
Comment (MancunianSam) [reply src/test/scala/uk/gov/nationalarchives/LambdaTest.scala:276]: Ok, I've updated the exception in the lambda code as well.

**Review by techncl — APPROVED** [review]
👍

**Pull request merged**